test/EndToEnd/tests/ServicesTest.ps1

```diff
@@ -703,6 +703,40 @@ function Test-BatchEventsApi
     Assert-True $result
 }
 
+function Test-ExecuteInitScriptsPerSolution
```

Comment:
This test doesn't verify the change here.

Installing EF will run the init script and it will stay loaded. Also if any other tests install this package they will also run it.

If there are failures in the ``entityframework.sqlservercompact`` install script they won't be caught here, install-package ignores those. So at the end GetInstalledPackages will return everything regardless of what happened for init.ps1 and install.ps1.